packages/formdata-event/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- Copy the design doc to the repo; expand the timing section.
```

Comment:
I would maybe just phrase this as "Add design doc" since nobody but us knows about the doc :)